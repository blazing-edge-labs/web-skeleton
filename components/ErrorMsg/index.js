import React from 'react'
import PropTypes from 'prop-types'

const ErrorMsg = ({ children }) => (
  <p>
    <i className="fa fa-exclamation-triangle" />
    {children}
  </p>
)

ErrorMsg.propTypes = {
  children: PropTypes.string.isRequired,
}

export default ErrorMsg
