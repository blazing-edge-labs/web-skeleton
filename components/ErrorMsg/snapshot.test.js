import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMsg from './';

describe('ErrorMsg component snapshot', () => {
  it('renders basic required data', () => {
    const tree = renderer.create(<ErrorMsg>Something went wrong.</ErrorMsg>);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
