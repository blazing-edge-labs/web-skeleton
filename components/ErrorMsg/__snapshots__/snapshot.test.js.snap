exports[`ErrorMsg component snapshot renders basic required data 1`] = `
<p>
  <i
    className="fa fa-exclamation-triangle" />
  Something went wrong.
</p>
`;
